explorer-api/src/mix_nodes/http.rs

```diff
@@ -7,13 +8,83 @@ use rocket_okapi::openapi_get_routes_spec;
 use rocket_okapi::settings::OpenApiSettings;
 
 pub fn mix_nodes_make_default_routes(settings: &OpenApiSettings) -> (Vec<Route>, OpenApi) {
-    openapi_get_routes_spec![settings: list]
+    openapi_get_routes_spec![
+        settings: list,
+        list_active_set,
+        list_inactive_set,
+        list_standby_set,
+        summary
+    ]
 }
 
 #[openapi(tag = "mix_nodes")]
 #[get("/")]
 pub(crate) async fn list(
     state: &State<ExplorerApiStateContext>,
 ) -> Json<Vec<PrettyDetailedMixNodeBond>> {
-    Json(state.inner.mix_nodes.get_detailed_mixnodes().await)
+    Json(state.inner.mixnodes.get_detailed_mixnodes().await)
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/active-set/active")]
+pub(crate) async fn list_active_set(
+    state: &State<ExplorerApiStateContext>,
+) -> Json<Vec<PrettyDetailedMixNodeBond>> {
+    Json(get_mixnodes_by_status(
+        state.inner.mixnodes.get_detailed_mixnodes().await,
+        MixnodeStatus::Active,
+    ))
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/active-set/inactive")]
```

Comment:
This sounds a bit strange, that the inactive set is on a path named `active-set`. I don't have a good alternative proposal though... Maybe just `set`?

Reply:
Yeah, not the best I agree. Originally I was thinking `/status/inactive` but `status` is also really overloaded. We can bug Claudia for the right terminology some time...